client/src/utils/formatBytes.js

```diff
@@ -0,0 +1,14 @@
+/**
+ * Converts number of bytes to a more human readable unit.
+ *
+ * @param {number} bytes -- Number of bytes
+ * @param {number} digits -- Number of decimal places to round to
+ * @returns {string} bytes represented in a format with units
+ */
+export default function formatBytes(bytes, digits = 2) {
+  if (bytes === 0) return '0 Bytes';
+  const units = ['Bytes', 'KiB', 'MiB', 'GiB', 'TiB', 'PiB', 'EiB', 'ZiB', 'YiB'];
+  const d = digits < 0 ? 0 : digits;
+  const mag = Math.floor(Math.log(bytes) / Math.log(1024));
+  return `${parseFloat((bytes / (1024 ** mag)).toFixed(d))} ${units[mag]}`;
```

Comment:
dis some @hallgchris python magick right here

Reply:
It was shamelessly taken from StackOverflow
cf7c298